go/kbfs/libkbfs/interfaces.go

```diff
@@ -139,6 +139,10 @@ type blockRetrieverGetter interface {
 	BlockRetriever() BlockRetriever
 }
 
+type settingsDBGetter interface {
```

Comment:
Does this need to be its own type instead of just being another method on the `Config` interface?

Reply:
no reason not to have it be its own type